tldr/semantic.py

```
@@ -203,12 +356,100 @@ def get_model(model_name: Optional[str] = None):
+            else:
+                try:
+                    _model = _MLXEmbedder(hf_name, mlx_batch=mlx_batch, device="cpu")
+                except RuntimeError:
+                    _model = SentenceTransformer(upstream, device="cpu", trust_remote_code=True)
+        elif backend == "mlx":
+            mlx_batch = 8
+            if model_name in SUPPORTED_MODELS:
+                mlx_batch = SUPPORTED_MODELS[model_name].get("mlx_batch", 8)
+            _model = _MLXEmbedder(hf_name, mlx_batch=mlx_batch, device=device)
+        else:
+            if device == "cpu":
```

Ruff (0.15.12)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)